1.0/api/bower.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "polymer-site",
+  "version": "0.0.0",
+  "homepage": "https://github.com/Polymer/docs",
+  "authors": [
+    "Eric Bidelman <ebidel@gmail.com>"
+  ],
+  "description": "Polymer website",
+  "keywords": [
```

Comment:
for the new site, let's remember to remove this and use the main site's bower.json